src/Calculator/Controls/MathRichEditBox.cpp

```diff
@@ -72,6 +72,7 @@ MathRichEditBox::MathRichEditBox()
     {
         throw Exception::CreateException(hr);
     }
+    this->GotFocus += ref new RoutedEventHandler(this, &MathRichEditBox::OnGotFocus);
```

Comment:
Should we remove the delegate combination somewhere else?

Reply:
Sorry, what do you mean by '*delegate combination*'?